ingest_file.py

```diff
@@ -1,19 +1,27 @@
 #!/usr/bin/env python
+"""
+Usage:
+    ingest_file.py <queue> <refdes> <method> <deployment> <filenames>
+
+"""
+
 import pika
-import sys
+import docopt
 
+options = docopt.docopt(__doc__)
 
-def ingest_files(queue, refdes, method, deployment, filenames):
+
+def ingest_files(_queue, _refdes, _method, _deployment, _filenames):
```

Comment:
why was it necessary to add an underscore? Note that `_refdes` is not used in the function.

Reply:
The underscores were in response to a PEP-8 error that those parameter shadowed variables  outside the scope.

Good catch on the _refdes not being used.